config/settings.yml

```diff
@@ -14,9 +14,6 @@ plugins:
     default: false
   qa_undo_vote_action_window:
     default: 10
-  qa_comments_default:
```

Comment:
Note that I'm not introducing migrations to drop this because we've decided to not make this backwards compatible with the old plugin. This is going to be Discourse's flavor of the Q&A plugin built on top of the fork.